packages/bananagram-common/src/example.ts

```diff
@@ -0,0 +1,54 @@
+type BanangramTokens = 'a' | 'b' | 'c';
+type WildBananagramTokens = BanangramTokens | '*';
+type ChessTokens = 'K' | 'Q' | 'p';
+
+type NumberOfTokensMap<Tokens extends string> = {
+  [key in Tokens]: number;
+};
+
+export const numberOfBanangramTokens: NumberOfTokensMap<BanangramTokens> = {
```

Comment:
Maybe all of this code could live inside of a test file somewhere? It seems to only be used for testing at the moment?

Reply:
removed example code b1d854c